src/client.ml

```diff
@@ -273,19 +273,14 @@ let with_transaction_or_error t f =
       Monitor.try_with_join_or_error ~here:[%here] (fun () -> f t))
 ;;
 
-let rec connect ?(tries = 5) ~host ~db ~user ~password ?port () =
+let rec connect ?(tds_version = Dblib.V72) ?(tries = 5) ~host ~db ~user ~password ?port ()
+  =
   try
     let conn =
       Dblib.connect
         ~user
-        ~password (* We have issues with anything higher than this *)
-        ~version:
-          Dblib.V70
-          (* Clifford gives FreeTDS conversion errors if we choose anything else,
-             eg:
-             ("Error(CONVERSION, \"Some character(s) could not be converted into
-             client's character set.  Unconverted bytes were changed to question
-             marks ('?')\")") *)
```

Comment:
are these comments not necessary anymore with respect to the character set?

Reply:
Yeah I suspect these are from before we added the text conversion stuff.